.buildkite/pipeline.yaml

```diff
@@ -0,0 +1,24 @@
+steps:
+  - label: ":eslint: Lint"
+    command:
+      - "yarn install"
```

Comment:
Looks like we get a modern Yarn version, so that's good. Is that controlled by the `node:10` Docker image?

Reply:
yea, iirc as of node 6 or 8 it's automatically included in the Docker images.